ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClient.java

```diff
@@ -72,19 +72,42 @@ void createTopic(
   Set<String> listTopicNames();
 
   /**
-   * [warn] synchronous call to get the response
+   * Synchronous call to get a one or more topic's description.
    *
    * @param topicNames topicNames to describe
+   * @return map of topic name to description.
    */
   Map<String, TopicDescription> describeTopics(Collection<String> topicNames);
 
   /**
-   * [warn] synchronous call to get the response
+   * Synchronous call to get the config of a topic.
+   *
+   * @param topicName the name of the topic.
+   * @return map of topic config if the topic is known, {@link Optional#empty()} otherwise.
+   */
+  Map<String, String> getTopicConfig(String topicName);
+
+  /**
+   * Synchronous call to write topic config overrides to ZK.
+   *
+   * <p>This will add additional overrides, and not replace any existing, unless they have the same
+   * name.
    *
-   * @param topicName topicNames to describe
+   * <p>Note: each broker will pick up this change asynchronously.
+   *
+   * @param topicName the name of the topic.
+   * @param overrides new overrides to add.
+   * @return {@code true} if any of the {@code overrides} did not already exist
    */
-  public TopicCleanupPolicy getTopicCleanupPolicy(String topicName);
+  boolean addTopicConfig(String topicName, Map<String, Object> overrides);
```

Comment:
Why Map<String, Object> here vs Map<String, String> throughout the rest of the interface?

Reply:
Makes the interface easier to use. Kafka props are usual passed as `Map<String, Object>`. The fact they need to be a String to pass to Kafka is an implementation detail, IMHO.